src/components/token/Tokens.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import ReactLoading from 'react-loading';
+import { Link } from 'react-router-dom';
+import colors from '../../index.scss';
+import TokenRow from './TokenRow';
+import {
+    shouldRenderCustomTokens
+} from '../../feature';
+
+/**
+ * Displays custom tokens in a table with pagination buttons and a search bar.
+ */
+class Tokens extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            tokens: [
+            ],
+            page: 1,
+            loaded: true,
+            hasAfter: false,
+            hasBefore: false,
+        }
+    }
+
+    render() {
+        const loadPagination = () => {
+            if (this.state.tokens.length === 0) {
+                return null;
+            } else {
+                return (
+                    <nav aria-label="Token pagination" className="d-flex justify-content-center">
+                        <ul className="pagination">
+                            <li ref="tokenPrevious" className={(!this.state.hasBefore || this.state.tokens.length === 0) ? "page-item mr-3 disabled" : "page-item mr-3"}>
+                                <Link to={this} className="page-link">Previous</Link>
+                            </li>
+                            <li ref="tokenNext" className={(!this.state.hasAfter || this.state.tokens.length === 0) ? "page-item disabled" : "page-item"}>
```

Comment:
You are handling `this.state.tokens.length === 0` in the first condition of the `if`, right?